security/pkg/nodeagent/sds/server.go

```diff
@@ -51,15 +49,11 @@ func NewServer(options *ca2.Options, workloadSecretCache ca2.SecretManager) (*Se
 	s := &Server{
```

Comment:
just a random thought: I was confused by the file path for a moment here - we don't ship a nodeagent anymore do we? I guess it's time we move the `sds` package a level up.

Reply:
Yep, a lot of cleanups needed here